src/dataaggregator/config/config.go

```diff
@@ -0,0 +1,66 @@
+package config
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/cloudfoundry-incubator/candiedyaml"
+)
+
+const (
+	DefaultLoggingLevel = "info"
+	DefaultPollInterval = 30
+)
+
+type LoggingConfig struct {
+	Level string `yaml:"level"`
+}
+
+var defaultLoggingConfig = LoggingConfig{
+	Level: DefaultLoggingLevel,
+}
+
+type PolicyPollerConfig struct {
+	PollInterval int `yaml:"poll_interval"`
+}
+
+var defaultPolicyPollerConfig = PolicyPollerConfig{
+	PollInterval: DefaultPollInterval,
+}
+
+type DbConfig struct {
+	PolicyDbUrl string `yaml:"policy_db_url"`
+}
+
+type Config struct {
```

Comment:
Do we need it to be nested? If we only have 1 config item per, just pull them up. Do the grouping once we have more than a few.